android/templates/build/file_provider_paths.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths xmlns:android="http://schemas.android.com/apk/res/android">
+	<external-path name="external_files" path="."/>
```

Comment:
Let's add support for all directories.  This is okay from a security perspective since the provider's "exported" attribute is set "false".  So, any Android intent providing a content: URI to another app has to be given specific read/write permissions for that one URI.

```xml
<root-path name="root" path="."/>
<files-path name="files" path="."/>
<cache-path name="cache" path="."/>
<external-path name="external" path="."/>
<external-files-path name="external_files" path="."/>
<external-cache-path name="external_cache" path="."/>
```

Also note that the above "root-path" is undocumented, but but it works-around a known issue on Huawei devices having a removable SD card where those devices (running a forked Android OS) causes an exception within Google's FileProvider class.  We can either do this or create our own custom file content provider.
http://stackoverflow.com/questions/39895579/fileprovider-error-onhuawei-devices